setup.py

```diff
@@ -30,8 +30,16 @@
 
         license=LICENSE,
 
-        packages=['resolwe_bio'],
-        include_package_data=True,
+        packages=find_packages(),
+        package_data={
+            'resolwe_bio': [
+                'descriptors/*.yml',
+                'processes/**/*.yml',
+                'tests/files/*',
+                'tools/*.py',
+                'tools/*.R',
+            ]
+        },
         zip_safe=False,
         install_requires=(
             "resolwe>=1.0.0",
```

Comment:
Hmm, I'm using single quotes?

Do you want me to change all double-quotes in `setup.py` to single quotes? If yes, I would rather do it in a separate commit.